lib/models/abstractWellsMessage.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+function copyProps(fromObj, toObj) {
+  Object.keys(fromObj).forEach((prop) => {
+    toObj[prop] = fromObj[prop];
+  });
+}
+
+class AbstractWellsMessage {
+  constructor(obj) {
+    copyProps(obj, this);
```

Comment:
you could consider using object.assign here instead of this helper
```suggestion
    Object.assign(this, obj);
```

Reply:
Actually `Object.assign` doesn't work cause you'd have to `this = Object.assign(this, obj)` and you can't assign to `this`